src/components/PageLayout.js

```diff
@@ -7,6 +7,10 @@ import { Dialog } from "@material-ui/core";
 
 const PageLayout = ({ children, showHomePageLink = true, pageTitle, forceDesktopLayout = false }) => {
 
+    useEffect(() => {
+        document.title = pageTitle !== undefined ? `${pageTitle} - NIJobs` : "NIJobs";
```

Comment:
This would fail if pageTitle = null, for example, since `null !== undefined`. You can just use the boolean version of the variable on the check
```suggestion
        document.title = pageTitle ? `${pageTitle} - NIJobs` : "NIJobs";
```